src/widgets/activity-group/model/hooks/useActivityGroups.ts

```diff
@@ -19,14 +21,30 @@ type UseActivityGroupsReturn = {
 export const useActivityGroups = (
   appletId: string,
 ): UseActivityGroupsReturn => {
-  const {} = useAppletDetailsQuery(appletId);
-
-  const builder = createActivityGroupsBuilder({
-    allAppletActivities: allActivityMocks,
-    appletId: 'apid1',
-    progress: progressMocks,
+  const {} = useAppletDetailsQuery(appletId, {
+    select: data => data.data.result,
   });
 
+  // const allAppletActivities = useMemo(() => {
```

Comment:
Can you revert this file, please?
We'd integrate it when all api is done. No need to keep commented code.